src/demo.js

```diff
@@ -5,35 +5,22 @@ import {
     annotation_display, parse_translations, create_context, format_messages
 } from './fluent';
 
-const translations = `// Try editing the translations here!
-
-hello-world = Hello, world!
-
-shared-photos =
-    { $user_name } { $photo_count ->
-        [0] hasn't added any photos yet
-        [one] added a new photo
-       *[other] added { $photo_count } new photos
-    }.
-
-liked-comment =
-    { $user_name } liked your comment on { $user_gender ->
-        [male] his
-        [female] her
-       *[other] their
-    } post.
+const translations = `
```

Comment:
This adds a leading newline to the example. Is this intentional?

Reply:
yes. I didn't find an easy way on the boundary between CSS and ace to get the padding-top right.